ui/tests/acceptance/client-history-test.js

```diff
@@ -292,7 +337,7 @@ module('Acceptance | clients history tab', function (hooks) {
     assert.equal(currentURL(), '/vault/clients/history', 'clients/history URL is correct');
     assert
       .dom(SELECTORS.emptyStateTitle)
-      .includesText('No start date found', 'Empty state shows no billing start date');
+      .includesText('start date found', 'Empty state shows no billing start date');
```

Comment:
for enterprise this reads "No billing start date found" and for OSS "No start date found" 

this was previously passing, but started failing for some reason? so only checking for the words shared between both versions

Reply:
Are you running Vault enterprise locally with the tests (assuming it's failing locally?). The tests run on whatever version of vault you have loaded so that could be why